### ISSUE TITLE
garage_wo_car_flag: add parking places from "other"

### ISSUE THREAD

**Issue by pro100olga** [excieve/dragnet#50]
Now flag is raised if:
- there is no vehicle in declaration
- there is an estate object of type "garage"

Suggestion: besides garages, add also "other" estate objects, which have in their description something about parking place or place for a car. Suggested patterns, based on the analysis of the data are:
(((м[і|е]с[т|ц])|(стоянк)|(схов)|(хранен)|(зберіг)|(парк))(.*)((авто)|(трансп)|(машин)))
(((авто)|(трансп)|(машин))(.*)((м[і|е]с[т|ц])|(стоянк)|(схов)|(хранен)|(зберіг)|(парк)))
(гараж)
(парк[у|о]в)
(парк[і|и]нг)
(парко(.*)м[і|е]с[т|ц])

**Comment by pro100olga** [comment]
Checked on a partial dataset (88K)
~60 mismatches, all True, but should be False

There is a vehicle in declaration:
nacp_00d0b1ec-db68-4b74-b480-94f777d2b1fb
nacp_00dab510-cefe-4f5b-bdcf-2c1da9e9292c
nacp_9901c610-6c3d-419e-bfba-931b291768df
nacp_99042d99-0e63-488a-96d0-f837a3850f00
nacp_9929ee56-3e0a-4a08-9e65-d7b2ad69d4a0

**Comment by dchaplinsky** [comment]
Well, the question here is should we count in vehicles that aren't owned by
declarant or family. Current implementation doesn't count them. Let me know
if you'd like to change that

**Comment by pro100olga** [comment]
Right, I have checked considering ownership. Now 35 mismatches, all True, but should be False:

 - there is a vehicle and a garage, but both are not owned by family or declarant
nacp_00d0b1ec-db68-4b74-b480-94f777d2b1fb 
nacp_99b48112-4584-4c09-a80c-150945ea3fb6

 - there is a vehicle and a garage, owned by family member, but the type of ownership is "third party"
nacp_99324156-5791-426e-9055-720648df44b9

 - no vehicle, but a garage - which is not owned by family or declarant
nacp_9993f800-13ca-4986-9bf6-23e1868f5af0
nacp_99b0fc94-1d11-4538-a183-9f4604e51737
